plugins/rest_api_plugin.py

```diff
@@ -64,6 +70,7 @@ def get_config_boolean_value(section, key, default_value):
 airflow_rest_api_plugin_http_token_header_name = get_config_string_value("rest_api_plugin", "REST_API_PLUGIN_HTTP_TOKEN_HEADER_NAME", "rest_api_plugin_http_token")
 airflow_expected_http_token = get_config_string_value("rest_api_plugin", "REST_API_PLUGIN_EXPECTED_HTTP_TOKEN", None)
 web_authentication_enabled = configuration.getboolean("webserver", "AUTHENTICATE")
+rbac_authentication_enabled = configuration.getboolean("webserver", "RBAC")
```

Comment:
Set default in the event that RBAC is not included.

Reply:
It is returning false in case RBAC is not set. Verified in both Python 2.x and 3.x